docs/trace/links-creation-with-new-activities/README.md

```diff
@@ -0,0 +1,156 @@
+# Creating new root activities that link to an existing activity: A Sample
+
+This sample shows how to create new root activities that
+[link](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md#links-between-spans)
+to an existing activity. This can be useful in a fan-out or batched operation
+situation when you want to create a new trace with a new root activity
+BEFORE invoking each of the fanned out operations, and at the same time
+you want each of these new traces to be linked to the original activity.
+
+To give an example, let's say that:
+
+- Service A receives a request for an operation that impacts 1000s of
+resources.
```

Comment:
suggest to use some other term than "resource", as it could be confusing given OTel has its own concept of Resource!

Reply:
Not sure if there's an alternative good term here. Based on your feedback, I have now added a clarification about the terminology.